troposphere/settings/local.py.j2

```diff
@@ -81,9 +75,7 @@ USE_LOGIN_SELECTION= False
 
 # The location from where static files are aggregated
 # ./manage.py collectstatic moves files from STATICFILES_DIRS to STATIC_ROOT
-STATICFILES_DIRS = [
-    ("theme", "{{ THEME_PATH }}/{{ THEME_NAME }}"),
-]
+STATICFILES_DIRS = []
```

Comment:
For now lets not worry about staticfiles and webpack generation..

In the future, there are some things we can do to streamline this process and make it easier to do things like run troposphere from `runsslserver` without crashing on serving assets/static pages